### PR TITLE
CC-155 - Fixed Button styling

### PR DESCRIPTION
**Pull request by GeReV**
Fixed height of Button to 40 pixels.

Also cleaned up the code a bit.

### CODE REVIEW

File: src/components/Button/Button.jsx [hunk 1]
```diff
@@ -1,68 +1,80 @@
...
-    if (this.props.loading) {
+    if (loading) {
+      const loaderDotStyle = (type === Variants.SECONDARY || type === Variants.TERTIARY) ?
+        { backgroundColor: '#000', opacity: '0.1' } :
+        null;
+
       contentComponent = <Loader dotStyle={loaderDotStyle} />;
     } else {
-      contentComponent = this.props.label || this.props.children;
+      contentComponent = children || label;
```
Comment (GeReV) [inline src/components/Button/Button.jsx]: `children` should _always_ take precedent.

File: src/components/Button/Button.jsx [hunk 2]
```diff
@@ -1,77 +1,80 @@
...
 
-    let contentComponent;
-    if (this.props.loading) {
-      contentComponent = <Loader dotStyle={loaderDotStyle} />;
-    } else {
-      contentComponent = this.props.label || this.props.children;
-    }
+    contentComponent = <Loader dotStyle={loaderDotStyle} />;
+  } else {
+    contentComponent = children || label;
```
Comment (GeReV) [inline src/components/Button/Button.jsx]: `children` should _always_ take precedent.

File: src/components/Button/Button.jsx [hunk 3]
```diff
@@ -1,76 +1,86 @@
...
+
+import { getDataAttrs } from '../../dataUtils';
+
 import type { Data } from '../../types';
 
-const variants = { PRIMARY: 'primary', SECONDARY: 'secondary', TERTIARY: 'tertiary' };
-const size = { SMALL: 'small' };
+import style from './style.scss';
+
+const Variants = { PRIMARY: 'primary', SECONDARY: 'secondary', TERTIARY: 'tertiary' };
```
Comment (kangax) [inline src/components/Button/Button.jsx]: we gotta do that annoying `Object.freeze` wrapping here

File: src/components/Button/Button.jsx [hunk 4]
```diff
@@ -1,76 +1,86 @@
...
+const Sizes = { SMALL: 'small' };
```
Comment (chelseawework) [inline src/components/Button/Button.jsx]: In the future, it might be a good idea to remove the "small" option. It is not in the Plasma design docs.

**Comment by chelseawework** [comment]
According to `continuous-integration/travis-ci/p`, this is failing flow. It should pass before it is merged.

File: src/components/Button/Button.jsx (hunk 4 above)
Comment (GeReV) [reply src/components/Button/Button.jsx]: I agree.

**Comment by GeReV** [comment]
Right! Thanks you!

**Pull request merged**